erigon-lib/kv/mdbx/kv_mdbx_test.go

```diff
@@ -1092,3 +1107,34 @@ func TestDB_BatchTime(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func BenchmarkDB_Get(b *testing.B) {
+	_db := BaseCaseDBForBenchmark(b)
+	table := "Table"
+	db := _db.(*MdbxKV)
+
+	// buffered so we never leak goroutines
+	err := db.Update(context.Background(), func(tx kv.RwTx) error {
+		return tx.Put(table, u64tob(uint64(1)), u64tob(uint64(1)))
+	})
+	if err != nil {
+		b.Fatal(err)
+	}
+
+	// Ensure data is correct.
+	if err := db.View(context.Background(), func(tx kv.Tx) error {
+		for i := 1; i <= b.N; i++ {
+			b.ResetTimer()
```

Comment:
Reset timer outside of loop - once is enough

Reply:
done